public/resources/core/js/controllers/user.controller.js

```diff
@@ -59,11 +61,47 @@
         vm.isAdmin = isAdmin;
         vm.isSeller = isSeller;
         vm.isAuthenticated = isAuthenticated;
-	vm.cambiaCesvar = cambiaCesvar;
-	$scope.cesvar = 'Potorro';
+	vm.orgsVisible = orgsVisible;
+	vm.orgsInvisible = orgsInvisible;
+	vm.loggedAsIndividual = loggedAsIndividual;
+	vm.switchSession = switchSession;
+	vm.switchToUser = switchToUser;
+	vm.showOrgList = showOrgList;
+	vm.showOrgs = false;
+	vm.hasAdminRole = hasAdminRole;
+
+	function hasAdminRole() {
+	    var org = User.loggedUser.organizations.find(x => x.id == vm.currentUser.id);
+	    return !loggedAsIndividual() || org.roles.findIndex(x => x.name == "Admin") > -1;
+	}
+
+	function loggedAsIndividual() {
```

Comment:
This method has a confuse name, is called loggedAsIndividual, but it returns false when your are logged as an individual